process/process_linux.go

```diff
@@ -49,6 +90,110 @@ type MemoryMapsStat struct {
 	Swap         uint64 `json:"swap"`
 }
 
+func (m *MemoryMapsStat) updateAttribute(fields []string) error {
+	if len(fields) < 2 {
+		return fmt.Errorf("update attribute not enough fields")
+	}
+	var v *uint64
+	switch fields[0] {
+	case "Size:":
+		v = &m.Size
+	case "Rss:":
+		v = &m.Rss
+	case "Pss:":
+		v = &m.Pss
+	case "Shared_Clean:":
+		v = &m.SharedClean
+	case "Shared_Dirty:":
+		v = &m.SharedDirty
+	case "Private_Clean:":
+		v = &m.PrivateClean
+	case "Private_Dirty:":
+		v = &m.PrivateDirty
+	case "Referenced:":
+		v = &m.Referenced
+	case "Anonymous:":
+		v = &m.Anonymous
+	case "Swap:":
+		v = &m.Swap
+	default:
+		/* ignore other fields */
```

Comment:
nit: single line comment can be made by using

```suggestion
		// ignore other fields
```